StripeFinancialConnections/StripeFinancialConnections/Source/Native/Consent/ConsentViewController.swift

```diff
@@ -111,6 +115,19 @@ class ConsentViewController: UIViewController {
         // this fixes an issue where presenting a UIViewController
         // on top of ConsentViewController would stop the dot animation
         consentLogoView?.animateDots()
+
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(appWillEnterForeground),
+            name: UIApplication.willEnterForegroundNotification,
+            object: nil
+        )
+    }
+
+    @objc private func appWillEnterForeground() {
```

Comment:
I like the straightforward naming!  UIApplication.willEnterForegroundNotification > appWillEnterForeground is easy to understand (vs. naming it something more-specific like "reactiveDots" or else)